styles.css

```diff
@@ -526,3 +533,177 @@
   justify-content: center;
   align-self: center;
 }
+
+.block {
+  display: none;
+}
+
+@media only screen and (min-width: 420px) {
+  .greetings {
+    font-size: 56px;
+  }
+
+  .title {
+    font-size: 24px;
+  }
+
+  .paragraph {
+    font-size: 16px;
+  }
+
+  .icon {
+    font-size: 22px;
+  }
+
+  .contact-header {
+    font-size: 32px;
+    line-height: 44px;
+  }
+}
+
+@media only screen and (min-width: 650px) {
+  .none {
+    display: none;
+  }
+
+  .project-flex {
+    flex-direction: row;
+    flex-wrap: wrap;
+    width: 100%;
+  }
+
+  .paragraph {
+    max-width: 500px;
+  }
+
+  .work-container {
+    width: 71%;
+    padding-left: 20px;
+  }
+
+  .project-container {
+    width: 49%;
+  }
+
+  .works {
+    flex-direction: row;
+    width: 100%;
+    align-content: space-between;
+  }
+
+  .header-flex {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .nav-list {
+    display: flex;
+    width: 60%;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .img {
+    background-position: right bottom -30%;
+    background-size: 180px;
+  }
+
+  .icon {
+    font-size: 25px;
+  }
+
+  .contact {
+    padding: 120px 80px 0;
+  }
+
+  .ul-list:hover {
+    font-size: 20px;
+    transition: 0.8s ease-out;
+    color: #fff;
+  }
+
+  .img-works {
+    height: auto;
+    width: 60%;
+  }
+}
+
+@media only screen and (min-width: 970px) {
+  .nav-list {
+    width: 40%;
+  }
+
+  .icon-ul {
+    position: absolute;
+    flex-direction: column;
+    left: 30px;
+  }
+
+  .list {
+    text-align: start;
+  }
+
+  .contact {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .contact-header {
+    text-align: left;
+    font-size: 30px;
+    line-height: 45px;
+    width: 45%;
+  }
+
+  .form {
+    margin-top: 0;
+    width: 50%;
+  }
+
+  .contact-button {
+    align-self: start;
+    font-size: 18px;
+  }
+
+  .greetings-container {
+    margin-left: 6%;
+  }
```

Comment:
- [ ] OPTIONAL: The greetings element seems to take not more than in the original design, so I suggest you give a width of about 50% for desktop view. Kindly, compare yours and the original design from the attached screenshot images.
![image](https://user-images.githubusercontent.com/64894927/121423017-7e783e00-c978-11eb-92ce-23ac30b8d374.png)   
![image](https://user-images.githubusercontent.com/64894927/121423098-9485fe80-c978-11eb-87cd-9db0fcb267d4.png)